test/deprecated/Math/log1p.good

```diff
@@ -0,0 +1,15 @@
+log1p.chpl:8: warning: log1p is no longer included by default, please 'use' or 'import' the 'Math' module to call it
```

Comment:
I am a bit confused by this warning; if the module is no longer included by default, shouldn't this program fail to compile?

Reply:
I didn't fully delete the symbols from AutoMath, just added a deprecation warning for them.  So they'll still be included by default, but they'll call the Math version under the covers.

Reply:
I updated the PR description to be a little more specific on that front, sorry!